composer/metrics/nlp.py

```diff
@@ -196,7 +196,15 @@ def compute(self) -> Tensor:
 
 class InContextLearningMetric(Metric):
 
-    def update(self, batch: dict, output_logits: torch.Tensor, labels: torch.Tensor):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.needs_batch = True
+
+    def update(self,
+               batch: dict,
+               output_logits: Optional[torch.Tensor] = None,
+               labels: Optional[torch.Tensor] = None,
+               outputs: Optional[torch.Tensor] = None):
         """Abstract interface for computing an in-context learning metrics.
```

Comment:
Leave a comment in the docstring that the args are changing and `output_logits` will be removed in v0.21